javaScript-week1/Goodboy-Oldboy.js

```diff
@@ -0,0 +1,14 @@
+// Goodboy-Oldboy
+
+const dogYearOfBirth = 2020;
+let dogYearFuture = 2030;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = false;
+
+if (shouldShowResultInDogYears === false) {
```

Comment:
The logic is inverted here. You check if `shouldShowResultInDogYears` is not true, which would mean to show the age in human years, but you do multiply it by 7, and show dog years.
Also `shouldShowResultInDogYears === false` can be written as `!shouldShowResultInDogYears` for shorter.